composer.json

```diff
@@ -189,6 +189,9 @@
         "drupal/google_analytics": "^4.0",
         "drupal/honeypot": "^2.0",
         "drupal/hook_event_dispatcher": "~3.2.0",
+        "drupal/imageapi_optimize": "^4.0",
+        "drupal/imageapi_optimize_binaries": "^1.0@beta",
+        "drupal/imageapi_optimize_resmushit": "^2.0@beta",
```

Comment:
What do  these do that imagemagick doesn't? These seems like just additional bloat